common/djangoapps/student/models.py

```diff
@@ -1344,6 +1344,38 @@ def is_paid_course(self):
 
         return False
 
+    def is_active_course(self, cert_status):
+        """
+        `cert_status` is the certificate status for this enrollment
+        Returns (True) if end date of this course is in the future
+
+        """
+        if self.course_overview.end is None:
+            return True
+
+        time_delta = datetime.now(UTC)
+        if self.course_overview.end is not None and self.course_overview.end > time_delta:
```

Comment:
`self.course_overview.end is not None` you don't need this condition here. It will be `not None` because you are checking `None` condition in above `if`.